packages/utils/src/cryptography.spec.ts

```diff
@@ -8,39 +7,37 @@ import {
   derive_packet_tag,
   iterate_hash,
   recover_iterated_hash,
-  create_tagged_mac
-} from './cryptography.js'
+  create_tagged_mac,
+  core_crypto_set_panic_hook
+} from '../../core/lib/core_crypto.js'
+
+core_crypto_set_panic_hook()
+
+import { webcrypto } from 'node:crypto'
+// @ts-ignore
+globalThis.crypto = webcrypto
 
 import {
   generateKeyShares,
-  Hash,
   iterateHash,
   PRG as TS_PRG,
   PRP as TS_PRP,
   recoverIteratedHash,
-  stringToU8a,
-  u8aEquals,
-  u8aToHex,
-  SECRET_LENGTH,
   createMAC,
   keyShareTransform as forwardTransform,
   derivePacketTag,
   derivePRGParameters,
   derivePRPParameters
-} from '@hoprnet/hopr-utils'
+} from './crypto/index.js'
 
 import assert from 'assert'
 
 import { createSecp256k1PeerId } from '@libp2p/peer-id-factory'
+import { stringToU8a, u8aEquals, u8aToHex } from './u8a/index.js'
+import { SECRET_LENGTH } from './constants.js'
+import { Hash } from './types.js'
 
 describe('cryptographic correspondence tests', async function () {
-  it('digest correspondence', async function () {
```

Comment:
intended to remove this basic test?

Reply:
Yes, the corresponding TS has been already removed